packages/examples/src/quickstart/README.md

```diff
@@ -1,10 +0,0 @@
-# Quick Start Things
-
-This set of Things can be used to build mashup applications for a smart home scenario.
-These Things are:
-
--   Simple coffee machine that can take an order to brew a coffee.
--   Presence sensor that emits an event when a person is detected.
--   Smart clock that runs 60 times faster than real time, where 1 hour happens in 1 minute.
-
-These Things are hosted on plugfest.thingweb.io but can be also self-hosted.
```

Comment:
BTW, maybe we can adjust the link to https://www.thingweb.io/services
@egekorkan ?

Reply:
I've adjusted the link accordingly in 5f123ea608346424d4bd055efd7b35c18e22ced3 :)